### ISSUE TITLE
logging.yaml not found

### ISSUE THREAD

**Issue by kensekense** [akb89/ortografix#1]
When I try to run the ortografix default command (with proper directories for data), I get this issue.
```
Traceback (most recent call last):
  File "/home/kense/.local/bin/ortografix", line 5, in <module>
    from ortografix.main import main
  File "/home/kense/.local/lib/python3.6/site-packages/ortografix/main.py", line 31, in <module>
    os.path.join(os.path.dirname(__file__), 'logging', 'logging.yml')))
  File "/home/kense/.local/lib/python3.6/site-packages/ortografix/utils/config.py", line 19, in load
    with open(config_file, 'r') as config_stream:
FileNotFoundError: [Errno 2] No such file or directory: '/home/kense/.local/lib/python3.6/site-packages/ortografix/logging/logging.yml'
```

**Comment by akb89** [comment]
how did you install the package? What is your environment (OS, python version, etc.)?

**Comment by kensekense** [comment]
I tried installing four separate times: once using pip, once via cloning and running the setup, and both again in a virtual environment.
I'm using Python 3 and I'm on Windows Subsystem for Linux.

**Comment by kensekense** [comment]
Okay, I managed to fix the issue by manually copying the logging.yml file into the proper location that it was looking for it in. It seems to be running now!

**Comment by akb89** [comment]
Should be fixed now with v0.3.1

**Comment by kensekense** [comment]
Thank you!
